packages/@coorpacademy-components/src/molecule/bulk-progress-bar/types.ts

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+export const propTypes = {
+  'data-name': PropTypes.string,
+  status: PropTypes.oneOf(['success', 'inProgress', 'fail']).isRequired,
+  progress: PropTypes.number.isRequired
+};
+
+export type Props = PropTypes.InferProps<typeof propTypes>;
```

Comment:
tu peux faire ca à la place comme ca tu pourras utiliser le type Status dans ton fichier index.tsx, et aussi tu pourras typer tes fixtures avec Props
```suggestion
export type Status = 'inProgress' | 'fail' | 'success';
export type Props = {
  data-name: string;
  status: Status;
  progress: string;
};
```

Reply:
on aura plus de runtime type checking avec cette solution. 🤔